include/zephyr/bluetooth/audio/gmap.h

```diff
@@ -0,0 +1,217 @@
+/** @file
+ *  @brief Header for Bluetooth Gaming Audio Profile (GMAP).
+ *
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_INCLUDE_BLUETOOTH_AUDIO_GMAP_
+#define ZEPHYR_INCLUDE_BLUETOOTH_AUDIO_GMAP_
+
+#include <zephyr/bluetooth/conn.h>
+#include <zephyr/sys/util_macro.h>
+
+/**
+ * @brief Bluetooth Gaming Audio Profile (GMAP)
+ * @defgroup bt_gmap Bluetooth Gaming Audio Profile
+ * @ingroup bluetooth
+ * @{
+ */
+
+/** Gaming Role bitfield */
+enum bt_gmap_role {
+	/** @brief Gaming Role Unicast Game Gateway
+	 *
+	 * Requires @kconfig(CONFIG_BT_CAP_INITIATOR), @kconfig{CONFIG_BT_BAP_UNICAST_CLIENT} and
+	 * @kconfig{CONFIG_BT_VCP_VOL_CTLR} to be enabled.
+	 */
+	BT_GMAP_ROLE_UGG = BIT(0),
+	/** Gaming Role Unicast Game Terminal
+	 *
+	 * Requires @kconfig(CONFIG_BT_CAP_ACCEPTOR) and @kconfig{CONFIG_BT_BAP_UNICAST_SERVER} to
+	 * be enabled.
+	 */
+	BT_GMAP_ROLE_UGT = BIT(1),
+	/** Gaming Role Broadcast Game Sender
+	 *
+	 * Requires @kconfig(CONFIG_BT_CAP_INITIATOR) and @kconfig{CONFIG_BT_BAP_BROADCAST_SOURCE}
+	 * to be enabled.
+	 */
+	BT_GMAP_ROLE_BGS = BIT(2),
+	/** Gaming Role Broadcast Game Receiver
+	 *
+	 * Requires @kconfig(CONFIG_BT_CAP_ACCEPTOR), @kconfig{CONFIG_BT_BAP_BROADCAST_SINK} and
+	 * @kconfig{CONFIG_BT_VCP_VOL_REND} to be enabled.
+	 */
+	BT_GMAP_ROLE_BGR = BIT(3),
+};
+
+/* TBD: Should we declare a single enum for feature supported, and then transform from the spec
+ * defined values to Zephyr define values
+ */
+
+/** Unicast Game Gateway Feature bitfield */
+enum bt_gmap_ugg_feat {
+	/** @brief Support transmitting multiple LC3 codec frames per block in an SDU
+	 *
+	 * Requires @kconfig{CONFIG_BT_BAP_UNICAST_CLIENT_ASE_SRC_COUNT} > 0
+	 */
+	BT_GMAP_UGG_FEAT_MULTIPLEX = BIT(0),
+	/** @brief 96 kbps source support
+	 *
+	 * Requires @kconfig{CONFIG_BT_BAP_UNICAST_CLIENT_ASE_SRC_COUNT} > 0
+	 */
+	BT_GMAP_UGG_FEAT_96KBPS_SOURCE = BIT(1),
+	/** @brief Support for receiving at least two channels of audio, each in a separate CIS
+	 *
+	 * Requires @kconfig{CONFIG_BT_BAP_UNICAST_CLIENT_ASE_SNK_COUNT} > 1 and
+	 * @kconfig{CONFIG_BT_BAP_UNICAST_CLIENT_GROUP_STREAM_COUNT} > 1
+	 */
+	BT_GMAP_UGG_FEAT_MULTISINK = BIT(2),
+};
+
+/** Unicast Game Terminal Feature bitfield */
+enum bt_gmap_ugt_feat {
+	/** @brief Source support
+	 *
+	 * Requires @kconfig{CONFIG_BT_ASCS_ASE_SRC_COUNT} > 0
+	 */
+	BT_GMAP_UGT_FEAT_SOURCE = BIT(0),
+	/** @brief 80 kbps source support
+	 *
+	 * Requires BT_GMAP_UGT_FEAT_SOURCE to be set as well
+	 */
+	BT_GMAP_UGT_FEAT_80KBPS_SOURCE = BIT(1),
+	/** Sink support
+	 *
+	 * Requires @kconfig{CONFIG_BT_ASCS_ASE_SNK_COUNT} > 0
+	 */
+	BT_GMAP_UGT_FEAT_SINK = BIT(2),
+	/** @brief 64 kbps sink support
+	 *
+	 * Requires BT_GMAP_UGT_FEAT_SINK to be set as well
+	 */
+	BT_GMAP_UGT_FEAT_64KBPS_SINK = BIT(3),
+	/** @briefSupport for receiving multiple LC3 codec frames per block in an SDU
```

Comment:
FWIW you may actually want to not even have `@brief` tags as it's implicit, just like e.g. you're not having it for the brief of the enum itself at L74

Reply:
Does doxygen always implicitly consider the first paragraph (i.e. until the first empty line) as the `@brief`? I was under the impression that `@brief` should be used for any multi-line documentation. 

Either way, using `@brief` (vs not using it) is more consistent with existing codebase, don't you agree?

Reply:
Yep, consistency is good! :) I guess my point was actually about consistency, since you already have a mix of `@brief` and implicit briefs for the enum and structs in this file, so would be good to go for one or the other indeed.

> Does doxygen always implicitly consider the first paragraph (i.e. until the first empty line) as the `@brief`? 

The brief is the contents of the first line until the first dot (if there is one)

> I was under the impression that `@brief` should be used for any multi-line documentation.

Ya, it was until https://github.com/zephyrproject-rtos/zephyr/pull/60314 -- I made this change to help in exactly this 👇 kind of situation, where if people have already gone as far as using Doxygen style comment, then there is a 99.9% chance that the text is indeed a brief description, even if does not have the `@brief` tag

```
/** Gaming Role bitfield */
enum bt_gmap_role {
```